app/templates/components/preprint-footer-branded.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="text-center">
+    <span>{{model.name}}:</span>
+    <div style="display: inline;">
+        <span>
+            <a href="mailto:{{if model.emailSupport model.emailSupport (concat 'contact+' model.id '@osf.io')}}"
+               onbeforeclick={{action 'click' 'link' 'Branded Footer - Support'}}
+            >
+                {{t "components.preprint-footer-branded.support"}}
+            </a>
+        </span>
+        <span> | </span>
+        <span>
+            <a href="mailto:{{if model.emailContact model.emailContact (concat 'support+' model.id '@osf.io')}}"
+               onbeforeclick={{action 'click' 'link' 'Branded Footer - Contact'}}
+            >
+                {{t "components.preprint-footer-branded.contact"}}
+            </a>
+        </span>
+    </div>
+    <div style="display: inline; margin-left: 8px;">
+        <a href="https://twitter.com/{{model.socialTwitter}}"
```

Comment:
I talked about this with Nici, and I think it was a requirement for the providers to have all three, but either way, I like your approach better. It should not be displayed if the model doesn't have the field.